### DISCUSSION TITLE
Github Copilot not showing suggestions in Visual Studio 2022

### DISCUSSION THREAD

**Discussion by sh4d30** [community/community#43867] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hey!

I'm unable to get Copilot (1.65.0.0) to give suggestions in Visual Studio Community 2022 (Version 17.4.3).

The extension (CTRL + ALT + Enter) appears to be stuck on "Loading Solutions..." and there is no errors in the Output. 

Has anyone experienced this?

Thanks!

**Comment by sh4d30** [comment, marked as answer]
Working after the Visual Studio update (Version 17.4.4)!

**Comment by DavidThielen** [comment]
It was working fine for me and now - same problem. So it's back...
